src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write code here
+const workersList = document.querySelector('ul');
+const workersArr = [...workersList.children];
+
+workersList.innerHTML = '';
+
+sortList(workersArr).forEach((worker) => workersList.append(worker));
+
+function getSalary(worker) {
+  const workerSalary = worker.getAttribute('data-salary').replaceAll('$', '');
```

Comment:
Ensure that the `data-salary` attribute is always formatted correctly with a dollar sign and commas, as the `replaceAll` method is used to remove these characters. If the format is inconsistent, consider adding error handling.